crates/matrix-sdk/src/room/timeline/event_handler.rs

```diff
@@ -679,6 +687,19 @@ fn _update_timeline_item(
     }
 }
 
+fn find_local_echo_by_event_id<'a>(
+    lock: &'a [Arc<TimelineItem>],
```

Comment:
Why `lock`?

Reply:
Copy-pasted from a place that was probably refactored from a lock guard to just a plain reference.